src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -0,0 +1,266 @@
+import sql from 'sql-template-tag'
+
+import { generateTestClient } from '../../../../utils/getTestClient'
+import { SetupParams, setupMSSQL } from '../../../../utils/setupMSSQL'
+
+describe('blog-env-mssql', () => {
```

Comment:
Nitpick: We should consider more descriptive names than blog-env-<dbname>(?). I had to adapt an old test called blog-env-postgres recently, but only the auto recovery part, and it was initially unclear from the test suite name/description what was actually being verified. Just something to think about for future tests :)

Reply:
I can change this, I agree.

Reply:
Done!